### PR TITLE
[v1.0] update common lib

### PR DESCRIPTION
**Pull request by XinyaoWa**

### CODE REVIEW

**Comment by xuechendi** [comment]
@tianyil1 @csdingbin @Peach-He , please take a look of this refine

**Comment by csdingbin** [comment]
@XinyaoWa  Could you did a test to see whether this PR can influence DE-NAS CNN and ViT training functionality?

File: e2eAIOK/common/trainer/utils/utils.py [hunk 1]
```diff
@@ -200,9 +274,126 @@ def create_metric(cfg):
...
+    ''' create optimizer
+
+    :param model: model, will be disabled if 'parameters' is not None
+    :param cfg: configurations
+    :param parameters: parameters for optimizer
+
+    :return: a optimizer
+    '''
+    print(F"model:{model}")
+    print(F"parameters:{parameters}")
```
Comment (Peach-He) [inline e2eAIOK/common/trainer/utils/utils.py:292]: please use logging to for easier log tracking

**Comment by XinyaoWa** [comment]
> 

denas test label has been added

**Comment by XinyaoWa** [comment]
> @XinyaoWa Could you did a test to see whether this PR can influence DE-NAS CNN and ViT training functionality?

Has added "denas test" label.

File: e2eAIOK/common/trainer/utils/utils.py (hunk 1 above)
Comment (XinyaoWa) [reply e2eAIOK/common/trainer/utils/utils.py]: have modified.

**Comment by csdingbin** [comment]
> > @XinyaoWa Could you did a test to see whether this PR can influence DE-NAS CNN and ViT training functionality?
> 
> Has added "denas test" label.

At present, denas label only covers the automatic test of search part, and the training part needs to be tested manually. Could you test the DE-NAS CV/ASR/NLP training functionality on your local server?

File: e2eAIOK/common/default.conf [hunk 2]
```diff
@@ -0,0 +1,74 @@
+## show all the configurations and default values in common lib
+## include required and optional settings
+
```
Comment (Peach-He) [inline e2eAIOK/common/default.conf:3]: all default config files of aiok are under e2eAIOK/conf folder, can we put this config file there?

**Review by Peach-He — APPROVED** [review]
LGTM

**Comment by xuechendi** [comment]
@XinyaoWa , @Peach-He , did you tested trainer for CNN/ViT/ASR/Bert as well? 
@zigzagcai, I just went through current tests triggered by github action, we still don't have Denas trainer included yet, right?

**Comment by zigzagcai** [comment]
ci for denas trainer has already been included

**Comment by xuechendi** [comment]
> ci for denas trainer has already been included.@xuechendi

@zigzagcai , I see CNN Denas is not including training, 
image

**Comment by xuechendi** [comment]
> > ci for denas trainer has already been included.@xuechendi
> 
> @zigzagcai , I see CNN Denas is not including training, image

and ViT doesn't have trainning?
image

**Comment by zigzagcai** [comment]
> > > ci for denas trainer has already been included.@xuechendi
> > 
> > @zigzagcai , I see CNN Denas is not including training, image
> 
> and ViT doesn't have trainning?
> image
> 

cnn/vit training test included in unittest.
asr/nlp training test included in ci integration test.

**Comment by xuechendi** [comment]
> > > > ci for denas trainer has already been included.@xuechendi
> > > 
> > > 
> > > @zigzagcai , I see CNN Denas is not including training, image
> > 
> > 
> > and ViT doesn't have trainning?
> > image
> 
> cnn/vit training test included in unittest. asr/nlp training test included in ci integration test.

@zigzagcai , I see, in that case, I'll merge this PR.
BTW, can you move  def test_cnn_trainer(self): and   def test_vit_trainer(self): to CI/CD as well. And for UT test, see if there is exception is not a good UT, normally, a result output check is required in UT. I'll add another issue so you can work on that

**Review by xuechendi — APPROVED** [review]
Codes are looking good, and check with @zigzagcai , current UT are covering all necessary path, will merge here

**Pull request merged**